Add lintr continuous integration (#96)
epiverse-trace/tutorials: episodes/simulating-transmission.Rmd

> Now we are ready to run our model. To install the `epidemics` package : 

Reviewer: ```suggestion
Now we are ready to run our model. Let's load the `epidemics` package : 
```

> ```{r installation, eval = FALSE, echo = 2:3}

Reviewer: ```suggestion
```{r loading, eval = FALSE, echo = 2:3}
```

Author: ```suggestion
```{r loading}
```

> # nolint start
> if (!require("pak")) install.packages("pak")
> pak::pak("epiverse-trace/epidemics")
> # nolint end

Reviewer: ```suggestion
library(epidemics)
```
